fe1-web/src/core/network/jsonrpc/__tests__/Greeting.test.ts

```diff
@@ -0,0 +1,104 @@
+import { mockAddress, mockChannel, mockKeyPair, mockPublicKey } from '__tests__/utils';
+import { ProtocolError, PublicKey } from 'core/objects';
+
+import { Greeting } from '../Greeting';
+
+const otherAddress = 'some other address';
+
+describe('Greeting', () => {
+  describe('constructor', () => {
+    it('can correctly construct a greeting instance', () => {
+      const peers = [otherAddress];
+
+      const g = new Greeting({
+        address: mockAddress,
+        channel: mockChannel,
+        peers,
```

Comment:
Is it intended that it's not `peers: peers` ?

Reply:
That is a shorthand notation for property names :) ("Object Property Value Shorthand") (It allows you to write `x` instead of `x:x` and is for example described here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#new_notations_in_ecmascript_2015)

Reply:
I didn't know about it, thanks !